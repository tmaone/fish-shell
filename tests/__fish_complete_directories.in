cd ..
mkdir test/data/abc
mkdir test/data/abcd
touch test/data/af
touch test/data/abcdf
mkdir test/data/xy
mkdir test/data/xyz
touch test/data/xyf
touch test/data/xyzf
echo \# __fish_complete_directories test/z
__fish_complete_directories test/z
echo \# __fish_complete_directories test/d
__fish_complete_directories test/d
echo \# __fish_complete_directories test/data
__fish_complete_directories test/data
echo \# __fish_complete_directories test/data/
__fish_complete_directories test/data/
echo \# __fish_complete_directories test/data/abc
__fish_complete_directories test/data/abc 'abc dirs'
