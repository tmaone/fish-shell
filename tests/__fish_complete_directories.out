# __fish_complete_directories test/z
# __fish_complete_directories test/d
test/data/	Directory
# __fish_complete_directories test/data
test/data/	Directory
# __fish_complete_directories test/data/
test/data/abc/	Directory
test/data/abcd/	Directory
test/data/fish/	Directory
test/data/xy/	Directory
test/data/xyz/	Directory
# __fish_complete_directories test/data/abc
test/data/abc/	abc dirs
test/data/abcd/	abc dirs
